Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2294,6 +2314,7 @@ public extension Tensor where Scalar: Numeric {
     ///   - exclusive: Indicates whether to perform an exclusive cumulative product.
     ///   - reverse: Indicates whether to perform the cumulative product in reversed order.
     /// - Returns: Result of the cumulative product operation.
+    /// - Precondition: `axis.rank` must be `0`.
```

Comment:
```suggestion
    /// - Precondition: `axis` must have rank `0`.
```

Reply:
Done in https://github.com/tensorflow/swift-apis/pull/614/commits/e1b6b405d96f412c4c284757ac823b6f1cf5534a.